harmonizer/src/lib.rs

```diff
@@ -55,13 +61,64 @@ pub fn harmonize_limit(
     // The snapshot is created in the build_harmonizer.rs script and included in our binary image
     let buffer = include_bytes!(concat!(env!("OUT_DIR"), "/composition.snap"));
 
+    let initial_heap_size =
+        std::env::var("APOLLO_HARMONIZER_EXPERIMENTAL_V8_INITIAL_HEAP_SIZE").unwrap_or_else(|_e| {
```

Comment:
Drive-by, I _think_ that this unused argument made the compiler upset and thus the failing CI tests. 😄  

```suggestion
        std::env::var("APOLLO_HARMONIZER_EXPERIMENTAL_V8_INITIAL_HEAP_SIZE").unwrap_or_else(|| {
```

Reply:
ah yes thank you, fixed it in a previous commit.